src/CommonITILObject.php

```diff
@@ -3835,6 +3851,17 @@ public static function dropdownStatus(array $options = [])
                 break;
         }
 
+        if ($p['use_template_limits'] !== false && (int) $p['use_template_limits'] > 0) {
```

Comment:
Item may have a status that is not allowed by template. Either because template has been updated, either because some hardcoded logic or rule has put a value that cannot be changed in UI. In this case, current status is not available, and preselected status in UI is the first of the list.
I guess current status should always be available.